languages/PLI.groovy

```diff
@@ -214,11 +244,19 @@ def createLinkEditCommand(String buildFile, LogicalFile logicalFile, String memb
 
 
 	// define the MVSExec command to link edit the program
+	if(isZUnitTestCase)
+	   parms = "DYNAM(DLL)"
```

Comment:
While we allow the parms to be file level overwrites, we should manage this property as such in the `samples/application-conf/PLI.properties` : 

`pli_linkEditParms=DYNAM(DLL) :: **/testcase/*.pli`
(https://github.com/IBM/dbb-zappbuild/blob/development/samples/application-conf/PLI.properties#L37)